swap/swap.go

```diff
@@ -229,7 +228,6 @@ func (s *Swap) sendCheque(peer enode.ID) error {
 	}
 
 	// reset balance;
-	// TODO: if sending fails it should actually be roll backed...
```

Comment:
Leave this one.

Reply:
I created an issue for this.

Reply:
#1643

Reply:
Thanks for the issue but I had requested it to be left in the code. Nevermind.